client_dashboard/common/src/commonMain/kotlin/org/thechance/common/domain/entity/TaxiFiltered.kt

```diff
@@ -0,0 +1,9 @@
+package org.thechance.common.domain.entity
+
+import org.thechance.common.domain.util.TaxiStatus
+
+data class TaxiFiltered(
```

Comment:
TaxiFiltered make it seem like the taxi is already filtered but this is the data class that give the parameters that filter the taxi so a better name for it is TaxiFilter

Reply:
but filter is verb